pkg/gofr/datasource/clickhouse/clickhouse.go

```diff
@@ -8,19 +8,19 @@ import (
 	"time"
 
 	"go.opentelemetry.io/otel/attribute"
-
 	"go.opentelemetry.io/otel/trace"
-
 	"github.com/ClickHouse/clickhouse-go/v2"
 )
 
+// Config holds the configuration needed to connect to the Clickhouse database.
```

Comment:
```suggestion
// Config holds the configuration needed to connect to the ClickHouse database.
```

This feedbacks could be applied everywhere, even on code you didn't changed (yet)

Reply:
hi, is it okay or should i update it ?